.github/path-ci/run-smoke.ps1

```
@@ -81,6 +81,44 @@ if (-not $found) {
 }
 Write-Host '[background] ok'
 
+Write-Host '== wt start =='
+$script:WtStartCaptured = $null
+function Start-Process {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Start-Process' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)